src/Components/Artwork/Metadata.tsx

```diff
@@ -54,6 +54,7 @@ export const Metadata: React.FC<MetadataProps> = ({
         hidePartnerName={hidePartnerName}
         hideArtistName={hideArtistName}
         isHovered={isHovered}
+        showHighDemandIcon={isMyCollectionArtwork}
```

Comment:
**suggestion(non-blocking):** maybe we should add a separate prop for the `showHighDemandIcon` instead of using `isMyCollectionArtwork`?

Reply:
@MrSltun I think that if we do that we will have to pass the prop all the way form Artwork component down here. But we already have `isMyCollectionArtwork` for which we want to show High Demand icon. That will be like doing the same thing twice, right? 🤔

Reply:
I understand your point, but my idea was about having this feature for non-MyCollection artworks, but that's not related to the current ticket :)

Reply:
Oh I see 👀👍🏾.  However, `Metadata.tsx` already uses `isMyCollectionArtwork` prop to determine the behaviour of the routerLink.